doc/man/bluechictl.1.md

```diff
@@ -56,6 +56,10 @@ Disable the list of systemd unit files for the `bluechi-agent`.
 **--no-reload**
     Don't reload daemon after disabling unit files
 
+### **bluechictl** *list-units-files* [*agent*]
+
+Fetches information about all systemd unit files on the bluechi-agents. If [bluechi-agent] is not specified, all agents are queried.
+
```

Comment:
Does the `--filter` options, which we have in `list-units` works also with `list-units-files`?

Reply:
It can work almost the same, filtering by path instead of unit ID. I commented it out because I wasn't sure it was supposed to but I'll add it back and add it to the man page